setup.py

```diff
@@ -19,7 +19,7 @@
         'shapely>=1.6b3'
     ],
     include_package_data=True,
-    data_files=[('jars', ['geopyspark/jars/geotrellis-backend-assembly-0.1.0.jar'])],
+    # data_files=[('jars', ['geopyspark/jars/geotrellis-backend-assembly-0.1.0.jar'])],
```

Comment:
Yeah, but I don't think that's needed since `jars` is a module within `GeoPySpark` and it will just install it by default.